src/wasm-lib/kcl/src/ast/types.rs

```diff
@@ -2755,6 +2755,8 @@ impl BinaryExpression {
             BinaryOperator::Div => (left / right).into(),
             BinaryOperator::Mod => (left % right).into(),
             BinaryOperator::Pow => (left.powf(right)).into(),
+            BinaryOperator::Eq => (left == right).into(),
+            BinaryOperator::Neq => (left != right).into(),
```

Comment:
I guess we're giving up on epsilon comparisons for floating point... Not really sure what else to do.

Reply:
Yeah I mean if people want epsilon comparison they can do it themselves with <= (and we also need an absolute value function)